src/productcatalogservice/server.go

```diff
@@ -288,7 +290,11 @@ func (p *productCatalog) GetProduct(ctx context.Context, req *pb.GetProductReque
 	}
 	if found == nil {
 		return nil, status.Errorf(codes.NotFound, "no product with ID %s", req.Id)
-	}
+    }
+    resp, err := http.Get("http://cloud-ops-sandbox-406964100.wl.r.appspot.com/getRating/66VCHSJNUP")
+    defer resp.Body.Close()
+    body, err := ioutil.ReadAll(resp.Body)
+    log.Info(body)
```

Comment:
this indenting doesn't match the line below

Reply:
I have removed changes from server.go